src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3287,7 +3309,7 @@ void MusicXMLParserDirection::direction(const String& partId,
         tempoLine->setBeginText(simplifiedText);
         tempoLine->setContinueText(u"");
         m_inferredTempoLineStart = tempoLine;
-    } else if (isLikelySticking() && isPercussionStaff) {
+    } else if (isLikelySticking()) {
```

Comment:
Makes `bool isPercussionStaff = m_pass1.isPercussionStaff(partId);`further up useless/unneeded/creating a compiler warning

Reply:
It's still used here:
handleDrumInstrument(isPercussionStaff, tick + m_offset);

Reply:
Ah, I missed that, thanks